include/Core/Posts/Modules/DeletePostsByCategoryModule.php

```diff
@@ -54,6 +54,7 @@ public function render() {
 			<table class="optiontable">
 				<?php
 				$this->render_filtering_table_header();
+				$this->render_attachment_settings();
```

Comment:
Let's show the checkbox after the Move/Delete ratio buttons.

Reply:
Moved delete attachment checkbox after Move/Delete radio buttons.